interpreter/valid/valid.ml

```diff
@@ -1,6 +1,7 @@
 open Ast
 open Source
 open Types
+open Values
```

Comment:
Nit: instead of a global` open`, maybe limit this one to a local `let open` inside is_const.

Reply:
Actually it looks like Values is used elsewhere in this file explicitly without `open` so I guess I can do it that way.